python/lsst/ts/observatory/control/utils/enums.py

```diff
@@ -30,17 +30,39 @@ class RotType(enum.IntEnum):
          to the North axis so rot_angle=0. means y-axis is aligned with North.
          Angle grows clock-wise.
 
+    SkyAuto: Same as sky position angle but it will verify that the requested
+             angle is achievable and wrap it to a valid range.
+
     Parallactic: This strategy is required for taking optimum spectra with
                  LATISS. If set to zero, the rotator is positioned so that the
                  y-axis (dispersion axis) is aligned with the parallactic
                  angle.
 
-    Physical_Sky: This strategy allows users to select the **initial** position
+    PhysicalSky: This strategy allows users to select the **initial** position
```

Comment:
For Parallactic (pre-existing issue) please clearly document that the angle starts out as specified, but then the instrument rotates with the sky.

In fact I'd be tempted to name this ParallacticSky to emphasize that, but that may be too disruptive.

Is this really the preferred mode? I can see arguments both ways. Rotating with the sky means that the position of the boresight doesn't matter. But if you care about the angle being Parallactic for any significant duration, then you lose that.